modules/land.js

```diff
@@ -8,11 +9,65 @@ function Land()
 }
 
 Land.prototype = {
-  draw: function(ctx)
+  draw: function()
```

Comment:
if you look at organism,

organism.draw(){
ctx [...]

this.drawVision(..);
this.drawRelationship(...);
}

this directly reflects what's in land.js

land.draw(){
ctx [..]

this.drawOrganisms()
this.drawFoods()
}